tests/src/ExistingSite/InstallStateTest.php

```diff
@@ -97,8 +97,8 @@ public function testConfig(): void {
     // Check page caching set for one year and css/js are aggregated.
     $performance_config = $this->config('system.performance');
     $this->assertSame(31536000, $performance_config->get('cache.page.max_age'));
-    $this->assertSame(TRUE, $performance_config->get('css.preprocess'));
-    $this->assertSame(TRUE, $performance_config->get('js.preprocess'));
+    $this->assertTrue(TRUE, $performance_config->get('css.preprocess'));
+    $this->assertTrue(TRUE, $performance_config->get('js.preprocess'));
```

Comment:
`assertTrue` does expect two condition, please use `$this->assertTrue($performance_config->get('css.preprocess'));` instead.

Reply:
done